services/web-app/components/full-page-error/full-page-error.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid, Link } from '@carbon/react'
+import PropTypes from 'prop-types'
+
+import styles from './full-page-error.module.scss'
+
+export const FullPageError = ({ title, subtitle, content, link, href }) => {
+  return (
+    <Grid>
+      <Column className={styles.column} sm={4} md={8} lg={{ start: 6, span: 6 }}>
```

Comment:
do we have an extra offset column here ? looks like figma is 4?
<img width="1046" alt="image" src="https://user-images.githubusercontent.com/40550942/180879028-d52af80d-732c-432f-8072-b07ad0919220.png">

<img width="1311" alt="image" src="https://user-images.githubusercontent.com/40550942/180878992-91a0f76a-e6e9-40f9-a91b-48cc0e0b0b6b.png">

Reply:
> Some comms from grid division, otherwise LGTM! 💃 should we get a design review on this? 👁️

yes,  smart!